services/opencollective/opencollective-by-tier.tester.js

```diff
@@ -1,11 +1,10 @@
 'use strict'
 
-const Joi = require('joi')
 const { nonNegativeInteger } = require('../validators')
 const t = (module.exports = require('../tester').createServiceTester())
 
 t.create('renders correctly')
-  .get('/shields/2988.json')
+  .get('/shields/2988.json?style=_shields_test')
```

Comment:
Several of the OpenCollective tests were missing the `_shields_test` style entirely but were silently ignoring the color check, the new helper highlighted this mistake.